client/basic-authorship/Cargo.toml

```diff
@@ -17,21 +17,23 @@ codec = { package = "parity-scale-codec", version = "1.3.4" }
 futures = "0.3.4"
 futures-timer = "3.0.1"
 log = "0.4.8"
-prometheus-endpoint = { package = "substrate-prometheus-endpoint", path = "../../utils/prometheus", version = "0.8.0"}
-sp-api = { version = "2.0.0", path = "../../primitives/api" }
-sp-runtime = { version = "2.0.0", path = "../../primitives/runtime" }
-sp-core = { version = "2.0.0", path = "../../primitives/core" }
-sp-blockchain = { version = "2.0.0", path = "../../primitives/blockchain" }
-sc-client-api = { version = "2.0.0", path = "../api" }
-sp-consensus = { version = "0.8.0", path = "../../primitives/consensus/common" }
-sp-inherents = { version = "2.0.0", path = "../../primitives/inherents" }
-sc-telemetry = { version = "2.0.0", path = "../telemetry" }
-sp-transaction-pool = { version = "2.0.0", path = "../../primitives/transaction-pool" }
+prometheus-endpoint = { package = "substrate-prometheus-endpoint", version = "0.8.0"}
+sp-api = { version = "2.0.0"}
+sp-runtime = { version = "2.0.0"}
+sp-core = { version="2.0.0"}
+sp-blockchain = { version = "2.0.0"}
+sc-client-api = { version = "2.0.0"}
+sp-consensus = { version = "0.8.0"}
+sp-inherents = { version = "2.0.0"}
+sc-telemetry = { version = "2.0.0"}
+sp-transaction-pool = { version = "2.0.0"}
```

Comment:
Why altered to use crates form repo version while we still have local ones?
If none of them are modified by us, we should remove them.

Reply:
basic authorship was taken from substrate repository where all of the pallets are avaialble locally - all of above are availavle in mangata repository